Formula/llvm.rb

```diff
@@ -142,6 +169,14 @@ def install
       lib.install_symlink "libLLVM.dylib" => "libLLVM-#{version.major}.dylib" unless build.head?
     end
 
+    on_linux do
+      # Strip executables/libraries/object files to reduce their size
+      system("strip", "--strip-unneeded", "--preserve-dates", *(Dir[bin/"**/*", lib/"**/*"]).select do |f|
+        f = Pathname.new(f)
+        f.file? && (f.elf? || f.extname == ".a")
+      end)
+    end
```

Comment:
Their build scripts generate an `install/strip` target. Wouldn't it be better to call that target instead on Linux? Or maybe calling it on both, though I'd want to know what this changes on macOS.